### ISSUE TITLE
etcd backup containers are failing to start on cloud

### ISSUE THREAD

**Issue by kdomanski** [kubermatic/kubermatic#1504]
```
kdomanski  ~  130  k -n kube-system get pods
NAME                                                             READY     STATUS                  RESTARTS   AGE
etcd-backup-2dp5fj857p-1530976324-krxf4                          0/1       Init:CrashLoopBackOff   530        1d
etcd-backup-5g7zrr4vsn-1530976182-lxd4x                          0/1       Init:CrashLoopBackOff   530        1d
etcd-backup-bbkhzmdcwm-1530977274-v78tt                          0/1       Init:CrashLoopBackOff   526        1d
etcd-backup-lg69pmx8wf-1530976337-v9q7p                          0/1       Init:CrashLoopBackOff   530        1d
etcd-backup-m6zvrkzlbt-1530976337-g5mss                          0/1       Init:CrashLoopBackOff   529        1d
etcd-backup-ms646rf4qk-1530976338-wj4jg                          0/1       Init:CrashLoopBackOff   530        1d
etcd-backup-q7wlmtwx57-1530977158-vb2qz                          0/1       Init:CrashLoopBackOff   527        1d
etcd-backup-rwhxp9j5j-1530976338-8qlll                           0/1       Init:CrashLoopBackOff   530        1d
etcd-backup-xt5684dfgf-1530976338-d7qjq                          0/1       Init:CrashLoopBackOff   529        1d
...
```

```
 kdomanski  ~  k -n kube-system describe pod etcd-backup-xt5684dfgf-1530976338-d7qjq
...
Events:
  Type     Reason   Age                  From                                                          Message
  ----     ------   ----                 ----                                                          -------
  Warning  BackOff  2m (x12084 over 1d)  kubelet, gke-kubermatic-cloud-europe-we-pool-1-f5d990f3-4n03  Back-off restarting failed container
```

**Comment by alvaroaleman** [comment]
This is caused by the fact that we introduced tls for etcd and the backup pods do not have the CA. This will get fixed within #1496

**Comment by alvaroaleman** [comment]
This was fixed in #1496
